native/src/test/scala/sqs4s/internal/CreateQueueSpec.scala

```diff
@@ -0,0 +1,43 @@
+package sqs4s.internal
+
+import java.time.Instant
+
+import cats.effect.{Clock, IO}
+import org.http4s.client.blaze.BlazeClientBuilder
+import org.scalatest.Ignore
+import sqs4s.api.SqsSetting
+import sqs4s.api.lo.CreateQueue
+import sqs4s.internal.aws4.IOSpec
+
+import scala.concurrent.duration.TimeUnit
+
+@Ignore
+class CreateQueueSpec extends IOSpec {
+  override implicit lazy val testClock: Clock[IO] = new Clock[IO] {
+    def realTime(unit: TimeUnit): IO[Long] = IO.delay {
+      Instant.now().toEpochMilli
+    }
+
+    def monotonic(unit: TimeUnit): IO[Long] = ???
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)